packages/sql/src/index.ts

```diff
@@ -101,13 +94,7 @@ const SqlPlugin: Plugin<AST | string> = {
           formatted = parser.sqlify(value, { type, database })
         }
 
-        // It can never be `auto`
-        // @see https://github.com/prettier/prettier/blob/ab72a2c11c806f3a8a5ef42314e291843e1b3e68/src/common/end-of-line.js#L3-L9
-        const ending = ENDINGS[endOfLine as keyof typeof ENDINGS]
-
-        formatted = formatted.replaceAll(/\r\n?|\n/g, ending)
-
-        return formatted.endsWith(ending) ? formatted : formatted + ending
+        return formatted + '\n'
```

Comment:
It doesn't read `endOfLine` option at all?

Reply:
When I tested it, EOL doesn't needed.

Reply:
It seems the tests do not cover `endOfLine` option which could be different with git source codes.

Reply:
Could you explain more about this?

As you can see the [link](https://github.com/un-ts/prettier/pull/358/files#diff-96eb36e78f73d856698b49a626ec0240b27ca6127b375c92e179130ca0ecb486)

my test case is covered `endOfLine` option

`559.sql` is marked `crlf` in `.gitattributes`. this case is intended to cover that people who using windows with `git.core.eol=auto`. becuase if the source code was commit with LF but those people will checkout with CRLF. so, in this case the formatter should print with CRLF as input.

through `556.sql` to `558.sql` is covered hardcoded eol case.

Reply:
OK, I read the snapshot wrong previous.

It seems `prettier` handles `endOfLine` option after plugin formatting? That's a bit strange to me.